src/Adapters/FHIRPatientAdapter.php

```diff
@@ -452,6 +465,22 @@ public function interfaceToModel( PatientInterface $patient )
         $fhirPatient->addTelecom( $email );
 
 
+        $address = new FHIRAddress();
+        $line = new FHIRString();
+        $line->setValue($patient->getStreet());
```

Comment:
Oh.
Now I understood. Thank you for detailed clarification.
Current functionality already returns array of street lines.
Do we need to add keys to address lines?
Like this 
"line": [
        [
          "address-line-1" : "Street Value 1",
          "address-line-2" : "Street Value 2"
        ]

![patient_adresses](https://cloud.githubusercontent.com/assets/5683856/19263128/73b620b8-8fa3-11e6-81c6-976b408680cc.png)

{
  "resourceType": "Patient",
  "extension": [
    {
      "extension": [
        {
          "valueString": "",
          "url": "#groupId"
        },
        {
          "valueString": "active",
          "url": "#status"
        },
        {
          "valueString": 34,
          "url": "#providerId"
        },
        {
          "valueString": 5,
          "url": "#pharmacyId"
        },
        {
          "valueString": "Some value of stripeToken",
          "url": "#stripeToken"
        }
      ],
      "url": "http://libre-emr.loc/fhir/extension/gponline-patient-data"
    }
  ],
  "identifier": [
    {
      "use": "usual",
      "value": 75
    }
  ],
  "name": [
    {
      "use": "usual",
      "family": [
        "Dow"
      ],
      "given": [
        "Judy Simple"
      ]
    }
  ],
  "telecom": [
    {
      "system": "phone",
      "value": "555-555-2003",
      "use": "primary"
    },
    {
      "system": "email",
      "value": "",
      "use": "primary"
    }
  ],
  "gender": "female",
  "birthDate": "1955-01-06",
  "address": [
    {
      "line": [
        [
          "address-line-1",
          "address-line-2"
        ]
      ],
      "city": "Sent-Petersburg",
      "district": "USA"
    }
  ]
}